src/main/kotlin/ui/view/timetracking/TimeTrackingView.kt

```diff
@@ -66,37 +128,26 @@ class TimeTrackingView : SuspendingIOSafeView("Gitlab Time Tracker") {
         bottom(TimeRecordingBarView::class)
```

Comment:
If `TimeRecordingBarView` is a suspending view, add borderpane extensions to UIScope